tools/ci_build/github/linux/docker/Dockerfile.ubuntu_openvino

```diff
@@ -1,9 +1,12 @@
 ARG UBUNTU_VERSION=20.04
-FROM ubuntu:${UBUNTU_VERSION}
-
 ARG OPENVINO_VERSION=2022.2.0
+FROM openvino/ubuntu20_runtime:${OPENVINO_VERSION} AS builder
```

Comment:
Where can we find the source code for openvino/ubuntu20_runtime, and how does it get updated if the image has any security vulnerability?

Reply:
https://github.com/openvinotoolkit/openvino . OpenVINO team is in charge of updating it in case of security vulnerability. The docker images are generated by OV CI/CD Pipeline.

Reply:
More specifically, would you mind to tell me where I can find the source code of this docker image, and how it was built?

Reply:
https://github.com/openvinotoolkit/docker_ci/blob/master/dockerfiles/ubuntu20/openvino_cgvh_runtime_2022.2.0.dockerfile

Reply:
The docker file is much more complicated than we have now. In this PR. you removed 10 lines, but added the dependency on the new docker file, which has 384 lines. I wonder if all the things are needed. For example, the docker file installs libavcodec(ffmpeg). We may need to buy licenses for the codecs. If you know, ffmpeg had a hall of shame page: https://ffmpeg.org/shame.html. Tencent was there, because one of their software used ffmpeg but they didn't follow the license requirements. I don't want to make the same mistake(which would put Microsoft on shame). So I need to know more details about the usage.

Reply:
In general, I prefer to not rely on external docker images.